ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -81,6 +83,8 @@ public MySqlAccountService(AccountServiceMetrics accountServiceMetrics, MySqlAcc
         throw e;
       }
     }
+    accountServiceMetrics.trackTimeSinceLastSync(this::getTimeInSecondsSinceLastSync);
+    accountServiceMetrics.trackContainerCount(this::getContainerCount);
```

Comment:
minor: how about moving these two to end  of constructor after cache is initialized and initial fetch is performed?

Reply:
I put them there so the gauge values (especially timeSinceLastSync would get populated during the initial fetch.